web/src/games/seabattle/board.tsx

```diff
@@ -79,7 +79,11 @@ export class Board extends React.Component<IBoardProps, IBoardState> {
       );
     }
     return (
-      <GameLayout optionsMenuItems={this._getOptionsMenuItems} gameArgs={this.props.gameArgs}>
+      <GameLayout
+        optionsMenuItems={this._getOptionsMenuItems}
+        gameArgs={this.props.gameArgs}
+        avoidOverscrollReload={true}
```

Comment:
Just a tip I learned: if you pass no value for a prop, it defaults to `true`.  So this can be just `avoidOverscrollReload`

Reply:
Good tip, thanks!